contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol

```diff
@@ -494,6 +525,11 @@ abstract contract KeeperRegistryBase2_1 is ConfirmedOwner, ExecutionPrevention {
     return transmitter.balance;
   }
 
+  function getTriggerType(uint256 upkeepId) public pure returns (Trigger) {
+    // TODO
+    return Trigger.CONDITION;
```

Comment:
just want to make sure this is intentional?

Reply:
for now at least...